uat/features/admin_menu.feature

```diff
@@ -23,7 +24,7 @@ Feature: Administration Menu
     And the "Account links" checkbox should be checked
     And the "Shortcuts" checkbox should not be checked
 
-  @api @javascript
+  @api @javascript @deploy @safe
```

Comment:
I'm tentatively OK with labeling clearing the cache a `@safe` operation, but I think we should do so intentionally. Could be bad on a large prod site at the wrong time.